### ISSUE TITLE
[FEATURE] Animation: Add / Remove tile at will

### ISSUE THREAD

**Issue by MrStevns** [Danik/Pyxel-Edit#37]
Recently a guy on the forum http://pyxeledit.com/forum/discussion/623/inserting-deleting-frames-in-an-animation#latest mentioned the lack of being able to remove or add a new tile at will. In the traditional sense of frames you can just add or remove a frame or in-between frame in the middle of an animation, potentially extending or shortening it. Atm. To do something similar, one would have to resize the canvas, change the number of tiles, select all your sprites, and manually move the sprites accordingly. This process could be make easier if it was possible choose exactly where you want to add or remove to the canvas width or height. 

Essentially it should be something similar to the pivot which is available in the resize document dialog, though here you have full control where you want the new tile to appear.

It should work like this:
You would use the mouse to select a spot + a given shortcut to add or remove an empty tile to the canvas. Currently i'm not sure how you would choose wether it should be added to the height or the width.. though i thought i would propose the idea, as i think it's an essential feature for animators.

**Comment by Danik** [comment]
Yes, this is a big problem with the current animation workflow. Another way of solving it would be to have a frame management tool, that can duplicate, insert or remove frames of the current animation, moving later frames in the canvas as necessary.
It could also solve the problem of copying or duplicating a frame containing multiple layers, which at the moment is a hassle since you need to copy it for each layer separately.
